src/value.jl

```diff
@@ -19,3 +19,12 @@ $(TYPEDEF)
 Abstract type of all values usable in constraints, including [`LinearValue`](@ref) and [`QuadraticValue`](@ref).
 """
 abstract type Value end
```

Comment:
Hmm this would open up ParameterValue right?

Reply:
if I get it right, yeah, abstract `Value` is there precisely for having stuff extensible

Reply:
yap, already did